dotnet/samples/KernelSyntaxExamples/Example45_MultiStreamingChatCompletion.cs

```diff
@@ -62,24 +60,25 @@ private static async Task OpenAIMultiStreamingChatCompletionAsync()
 
         var consoleLinesPerResult = 10;
 
-        ClearDisplayByAddingEmptyLines();
+        // Uncomment this if you want to use a console app to display the results
+        // ClearDisplayByAddingEmptyLines();
 
         var prompt = "Hi, I'm looking for 5 random title names for sci-fi books";
 
         await ProcessStreamAsyncEnumerableAsync(chatCompletionService, prompt, executionSettings, consoleLinesPerResult);
 
-        Console.WriteLine();
+        this._output.WriteLine();
 
         // Set cursor position to after displayed results
-        Console.SetCursorPosition(0, executionSettings.ResultsPerPrompt * consoleLinesPerResult);
+        // Console.SetCursorPosition(0, executionSettings.ResultsPerPrompt * consoleLinesPerResult);
```

Comment:
Is it still needed? There's also commented out code in lines 64, 92 and 108.

Reply:
I leave it there because if a sk developer wants to use this example in a console application, this line will make it better to show the streaming behavior.